src/EventTeachingService.cpp

```diff
@@ -154,9 +154,8 @@ Processed EventTeachingService::handleUnlearnEvent(const VlcbMessage *msg, unsig
   {
     if (msg->len < 5)
     {
-      controller->sendGRSP(OPC_EVULN, getServiceID(), CMDERR_INV_CMD);
       controller->sendCMDERR(CMDERR_INV_CMD);
-
+      controller->sendGRSP(OPC_EVULN, getServiceID(), CMDERR_INV_CMD);
```

Comment:
Swapped these error messages so that we consistently produce CMDERR before GRSP. This makes testing easier.